prover/protocol/distributed/compiler/permutation/permutation.go

```diff
@@ -0,0 +1,51 @@
+package permutation
+
+import (
+	"github.com/consensys/linea-monorepo/prover/protocol/coin"
+	"github.com/consensys/linea-monorepo/prover/protocol/compiler/permutation"
+	"github.com/consensys/linea-monorepo/prover/protocol/query"
+	"github.com/consensys/linea-monorepo/prover/protocol/wizard"
+)
+
+func IntoGrandProduct(comp *wizard.CompiledIOP) {
```

Comment:
Changes considering that we will skip the preparation phase:

AddGdProductQuery(initialComp, moduleComp, disc, targetModuleName)